api/specs/web-server/_users.py

```diff
@@ -45,6 +50,25 @@ async def update_my_profile(_profile: ProfileUpdate):
     ...
 
 
+@router.get(
+    "/me/preferences",
+    response_model=Envelope[FrontendUserPreferencesGet],
+)
+async def get_user_preferences():
+    ...
+
+
+@router.patch(
+    "/me/preferences/{frontend_preference_name}",
```

Comment:
actually, would this not be "/me/preferences/{preference_id}" ?
the fact that you only allow to change frontend preference is your responsability right? in case the frontend tries to change s4l preference then it should return Forbidden